### PR TITLE
adding aio-max-nr setting instructions to Docs 

### PR DESCRIPTION
**Pull request by guy9**
Adding aio-max-nr setting instructions to Docs and to the docker image description

Solves issue #15072

### CODE REVIEW

**Comment by tchaikov** [comment]
@guy9 hi Guy, you might need to add a backport label, so that the bot can backport the change if necessary, if you don't want to backport it, please add the "backport/none" label instead.

File: docs/dev/docker-hub.md [hunk 1]
```diff
@@ -40,6 +40,18 @@ to calculate the proper value is:
...
+For example, to check the existing value and update it to 1048576: 
+
+```console
+ubuntu $ cat /proc/sys/fs/aio-max-nr
+65536
+ubuntu $ echo "fs.aio-max-nr = 1048576" >> /etc/sysctl.conf
+ubuntu $ sysctl -p /etc/sysctl.conf
+fs.inotify.max_user_watches = 524288
+fs.aio-max-nr = 1048576
+ubuntu $
```
Comment (tchaikov) [inline docs/dev/docker-hub.md:52]: @lersek in case you are interested.

**Comment by tchaikov** [comment]
see also #5981 (Improve error message where fs.aio-max-nr is too low)

**Comment by lersek** [comment]
@tchaikov, thanks for the CC!

@mykaul, referring to our discussion that started [here](https://github.com/scylladb/scylladb/pull/20307#issuecomment-2312675269): this PR, i.e., #19921, (unintentionally) seems to justify my proposal to remove the stale language on `aio-max-nr` from `docs/dev/docker-hub.md`. Namely, in the commit that @guy9 proposes here, the same stale / otherwise incorrect language is being perpetuated in `docs/operating-scylla/procedures/tips/best-practices-scylla-on-docker.rst`.

@guy9, please see my list of problems with the language that you are duplicating. We either need to remove the original (and not copy it here), or we need to meticulously update the original (as I sought to lay out in the second part of [this comment](https://github.com/scylladb/scylladb/issues/5981#issuecomment-2291900412)), and then reference -- preferably: not copy -- that, at any new location.

**Comment by lersek** [comment]
@guy9 please help review #20398, and then please redo your PR on top of that. Thanks!

**Comment by guy9** [comment]
Closing this, replaced with PR #21627

**Pull request closed without merging**